alerting/delete.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Delete notification alerts
+
+function installing_alerting() {
+  echo Deleting custom alerts
+  kubectl delete -f custom-alerts/
+
+  echo Removing patch from Prometheus
+  kubectl patch Prometheus rancher-monitoring-prometheus -n cattle-monitoring-system --patch-file patch-cluster-name.yaml --type=merge
```

Comment:
instead of hardcoding namespace you can set it as an env variable.

Reply:
updated